sdk/formrecognizer/ai-form-recognizer/samples/javascript/copyModel.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how to copy a model from a source Form Recognizer resource
+ * to a target Form Recognizer resource.
+ */
+
+//const { FormTrainingClient, AzureKeyCredential } = require("@azure/ai-form-recognizer");
```

Comment:
should this be commented out?

Reply:
Good catch!  Fixed in https://github.com/Azure/azure-sdk-for-js/pull/9068/commits/16cf587eca692932cd59580f7f24d13d5449d61a